bioptim/dynamics/configure_problem.py

```diff
@@ -140,13 +140,34 @@ def torque_driven(
         ConfigureProblem.configure_qdot(nlp, True, False)
         ConfigureProblem.configure_tau(nlp, False, True, fatigue)
 
-        if implicit_dynamics:
+        if multibody_dynamics == MultiBodyDynamics.SEMI_EXPLICIT:
```

Comment:
Please make the order EXPLICIT', IMPLICIT, SEMI as the former is the one we will use the most (so it make sense it is firts

Reply:
After reviewing, it seems that EXPLICIT is not included in this, so IMPLICIT, SEMI